lib/iscsi-command.c

```diff
@@ -331,6 +339,11 @@ static void parse_sense_descriptors(struct scsi_sense *sense, const uint8_t *sb,
 		if (!(p[2] & 0x80)) /* VALID bit */
```

Comment:
Do you agree that not all descriptors have a VALID bit and hence that this if-statement should be moved inside the switch-statement?

Reply:
Good catch, it looks like `VALID` isn't actually present for `Sense key specific sense data descriptor`, at least not SPC-5 r17 . I'll split this out as a separate fix.